docs/release/trg-0/trg-9-03.md

```diff
@@ -0,0 +1,255 @@
+---
+title: TRG 9.03 - KIT Artifacts
+---
+
+| Status     | Created      | Post-History                           |
+|------------|--------------|----------------------------------------|
+| Draft      | 19-Feb-2024  | Initial contribution                   |
+
+This explains each artifact a KIT can or must follow - depending on the graduation stage (TRG 8.02). An artifact is textual content or a technical component that is part of a KIT (e.g. a Vision statement).
```

Comment:
The TRG referenced is wrong. Also a link is missing

Reply:
Updated.